stdlib/3/ssl.pyi

```diff
@@ -237,6 +248,9 @@ if sys.version_info >= (3, 5):
         context = ...  # type: SSLContext
         server_side = ...  # type: bool
         server_hostname = ...  # type: Optional[str]
+        if sys.version_info >= (3, 6):
+            session = ...  # type: Optional[SSLSession]
+            session_reused = ...  # type: bool
```

Comment:
This can not return `None` while the property above can... https://github.com/python/cpython/blob/5fe59f8e3a0a56a155c18f9d581205ec533764b6/Modules/_ssl.c#L2476